src/main/java/dev/mlml/command/impl/Echo.java

```diff
@@ -25,8 +25,8 @@ public Echo() {
                         .get(),
                 new StringArgument.Builder("text")
                         .description("The text to echo")
-                        .require()
                         .isVArgs()
+                        .defaultValue("Echooo")
```

Comment:
Same comment from the last PR:
Still think that this should be required, at least for the purposes of testing the parser